docs/_data/menu-sql.yaml

```diff
@@ -0,0 +1,79 @@
+- text: Getting Started
+  url: sql-getting-started.html
+  subitems:
+    - text: "Starting Point: SparkSession"
+      url: sql-getting-started.html#starting-point-sparksession
+    - text: Creating DataFrames
+      url: sql-getting-started.html#creating-dataframes
+    - text: Untyped Dataset Operations
```

Comment:
how about `Untyped Dataset Operations (DataFrame operations)`

Reply:
make sense, keep same with `sql-getting-started.html#untyped-dataset-operations-aka-dataframe-operations`, done in b3fc39d.